src/CalendarApp.Console/Presenters/DeleteMeetingPresenter.cs

```diff
@@ -0,0 +1,51 @@
+using CalendarApp.Console.Presenters.Interfaces;
+using CalendarApp.Domain.Services.Interfaces;
+using CalendarApp.Domain.Exceptions;
+using System.Linq;
+
+
+namespace CalendarApp.Console.Presenters;
+class DeleteMeetingPresenter : IPresenter
+{
+    private readonly IMeetingsService _meetingService;
+
+    public DeleteMeetingPresenter(IMeetingsService meetingsService)
+    {
+        _meetingService = meetingsService;
+    }
+    public void Show()
+    {
+        Clear();
+        WriteLine("Enter Meeting Name to delete: ");
+    }
+
+    public IPresenter Action()
+    {
+        static void pressAnyKey()
```

Comment:
code convention violated
```suggestion
        static void PressAnyKey()
```